timesketch/lib/sigma_util.py

```diff
@@ -66,6 +68,10 @@ def get_sigma_config_file(config_file=None):
 
     sigma_config = sigma_configuration.SigmaConfiguration(sigma_config_file)
 
+    if not sigma_config:
+        raise ValueError(
+            'sigma_config is none - Error')
```

Comment:
maybe have a better error message? also no need to break this into several lines for a short message like this

Reply:
Good point, actually a ```try``` block might be even better.